modules/drivers/druid/src/metabase/driver/druid/client.clj

```diff
@@ -81,7 +88,10 @@
       (log/warn (trs "Client closed connection, canceling Druid queryId {0}" query-id))
       (try
         (log/debug (trs "Canceling Druid query with ID {0}" query-id))
-        (DELETE (details->url details-with-tunnel (format "/druid/v2/%s" query-id)))
+        (DELETE (details->url details-with-tunnel (format "/druid/v2/%s" query-id))
+          :auth-enabled  (details :auth-enabled)
```

Comment:
Hi and thank you for the pr!! Here it is not immediately evident that `details` will be a map (and not nil), so I would suggest to use the keyword as a function instead of the (most likely) map like so: `(:auth-enabled details)`. this way we can avoid an null pointer error if `details` is nil.

Reply:
Thanks, I totally agreed. Fixed!